nixos/modules/services/web-apps/matomo.nix

```diff
@@ -136,7 +136,7 @@ in {
         UMask = "0007";
         # TODO: might get renamed to MATOMO_USER_PATH in future versions
         # chown + chmod in preStart needs root
-        PermissionsStartOnly = true;
+        StateDirectory = "matomo";
```

Comment:
Won't this break the preStart?

Reply:
I'm pretty sure it's all working I've been using it for a few days and I setup a VM and went through the install process to make sure that still worked.

Reply:
Looks to me like root is potentially required, but unable to determine without looking at previous module. Depends on whether or not you have a `/var/lib/piwik` folder.

This is the sorta thing `stateVersion` is for, I guess. Generally we shouldn't do things like this and leave it up to the user if a user name or folder name is really bugging them that much... :man_shrugging:

Reply:
I've removed it for now to address at a different time.